Description: omit failing tests
 Requires profile debug.
Author: Jonas Smedegaard <dr@jones.dk>
Forwarded: not-needed
Last-Update: 2023-08-14
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/tests/tests.rs
+++ b/tests/tests.rs
@@ -33,6 +33,7 @@
 }
 
 #[test]
+#[cfg(feature = "broken_test")]
 fn client() {
     let rc = client_command()
         .arg("https://google.com")
@@ -44,6 +45,7 @@
 
 #[cfg(feature = "acceptor")]
 #[test]
+#[cfg(feature = "broken_test")]
 fn server() {
     let mut srv = server_command()
         .arg("1337")
@@ -76,6 +78,7 @@
 }
 
 #[test]
+#[cfg(feature = "broken_test")]
 fn custom_ca_store() {
     let mut srv = server_command()
         .arg("1338")
